src/EnergyPlus/HeatingCoils.cc

```diff
@@ -2342,9 +2342,13 @@ namespace HeatingCoils {
                     OutletAirHumRat = FullLoadOutAirHumRat;
                     OutletAirTemp = FullLoadOutAirTemp;
                 } else {
-                    OutletAirEnthalpy = PartLoadRat * FullLoadOutAirEnth + (1.0 - PartLoadRat) * InletAirEnthalpy;
-                    OutletAirHumRat = PartLoadRat * FullLoadOutAirHumRat + (1.0 - PartLoadRat) * InletAirHumRat;
-                    OutletAirTemp = PartLoadRat * FullLoadOutAirTemp + (1.0 - PartLoadRat) * InletAirDryBulbTemp;
+                    OutletAirEnthalpy =
+                        PartLoadRat * AirMassFlow / HeatingCoil(CoilNum).InletAirMassFlowRate * (FullLoadOutAirEnth - InletAirEnthalpy) +
```

Comment:
So this was calculated as `h_out = (PLR)*h_out_full_load + (1-PLR)*h_in`, which essentially scales the enthalpy as a weighted fraction of outlet enthalpies for the time when the unit was on or off based on part load.  This seems to make sense if it was runtime fraction, but not with PLR.

This is now calculated as `h_out = h_in + PLR * (AirMassFlow/InletAirMassFlow) * (h_out_full_load - h_in)`, which is more of an enthalpy balance application and seems to make more sense.